src/Config.cpp

```diff
@@ -247,6 +247,12 @@ parse_compiler_type(const std::string& value)
     return CompilerType::nvcc;
   } else if (value == "other") {
     return CompilerType::other;
+  } else if (value == "cl") {
+    return CompilerType::cl;
+  } else if (value == "clangcl" || value == "clang-cl") {
+    // Allow also "clang-cl" because that's how the compiler is called
+    // but clang-cl is not a valid C++ name.
```

Comment:
No aliases here, please. This is what a human would enter in the configuration so one is enough.

Also, documentation is missing.

Reply:
So do you prefer just `clangcl` and not matching the compiler name, or `clang-cl`, and not using `CASE(clangcl)` in `compiler_type_to_string()`?

Reply:
I prefer `clang-cl` if we must since that seems to be its name, but I'd rather classify `clang-cl` as `cl` if possible as discussed above.